src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/audit/AuditInterceptor.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.audit
+
+import org.aspectj.lang.ProceedingJoinPoint
+import org.aspectj.lang.annotation.Around
+import org.aspectj.lang.annotation.Aspect
+import org.springframework.core.annotation.Order
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.repository.StoredProcedureRepository
+
+@Component
+@Aspect
+@Order(1)
```

Comment:
Why order 1 and not 0?

Reply:
I assumed 1 was the lowest number, and it fixed the actually transaction issue I predicted

Reply:
BTW looks like any number below the default of Integer.MAX_VALUE would have worked